logstash-core/src/test/java/org/logstash/config/ir/IRHelpers.java

```diff
@@ -0,0 +1,139 @@
+package org.logstash.config.ir;
+
+import org.hamcrest.MatcherAssert;
+import org.logstash.config.ir.expression.BooleanExpression;
+import org.logstash.config.ir.expression.ValueExpression;
+import org.logstash.config.ir.expression.unary.Truthy;
+import org.logstash.config.ir.graph.Edge;
+import org.logstash.config.ir.graph.Graph;
+import org.logstash.config.ir.graph.Vertex;
+import org.logstash.config.ir.graph.algorithms.GraphDiff;
+
+import java.util.HashMap;
+import java.util.Objects;
+import java.util.UUID;
+
+import static org.logstash.config.ir.DSL.*;
+import static org.logstash.config.ir.PluginDefinition.Type.*;
+
+/**
+ * Created by andrewvc on 9/19/16.
+ */
+public class IRHelpers {
+    public static void assertSyntaxEquals(SourceComponent left, SourceComponent right) {
```

Comment:
I just changed `assertGraphEquals` to an overloaded version of `assertSyntaxEquals`. Good call :)